tmgmt_lilt.info.yml

```
@@ -1,6 +1,6 @@
 name: Lilt Translator
 description: 'The Lilt translator module that integrates with the Translation Management Tool.'
-core_version_requirement: ^8.8 || ^9
+core_version_requirement: ^9  || ^10
```

should we still support 8.8 here?

```suggestion
core_version_requirement: ^8.8 || ^9  || ^10
```

---

Sure, I will update and test it